File: app/src/main/java/com/letianpai/robot/wificonnet/parser/BindData.java
package com.letianpai.robot.wificonnet.parser;

public class BindData {
    private boolean is_bind;
    private String country;

    public boolean isIs_bind() {
        return is_bind;
    }

    public void setIs_bind(boolean is_bind) {
        this.is_bind = is_bind;
    }

    public String getCountry() {
        return country;
    }

    public void setCountry(String country) {
        this.country = country;
    }

    @Override
    public String toString() {
        return "{" +
                "is_bind:" + is_bind +
                ", country:'" + country + '\'' +
                '}';
    }
}

File: app/src/main/java/com/letianpai/robot/wificonnet/parser/BindData.kt
package com.letianpai.robot.wificonnet.parser

class BindData {
    var isIs_bind: Boolean = false
        private set
    var country: String? = null

    fun setIs_bind(is_bind: Boolean) {
        this.isIs_bind = is_bind
    }

    override fun toString(): String {
        return "{" +
                "is_bind:" + isIs_bind +
                ", country:'" + country + '\'' +
                '}'
    }
}
